packages/test-runner/test/assets/nested-fixtures.js

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
@mxschmitt I took your test from #73. Is that ok?

Reply:
Thats awesome!